tests/transactional/install_updates.pm

```diff
@@ -16,6 +16,19 @@ use version_utils 'is_sle_micro';
 use serial_terminal;
 use utils qw(script_retry);
 
+sub soft_fail_rt_scriptlet {
+    return if (get_var('FLAVOR') !~ /rt/i);
+
+    if (script_run("grep '%post(kernel-rt-5.14.21-150400.15.46.1.x86_64) scriptlet failed, exit status 1' /var/log/zypp/history") == 0) {
```

Comment:
Can we avoid having the kernel version hard-coded?

Reply:
We can, but this time it is really specific to this one patch. Therefore, it appears only in slem 5.4 and 5.3. It is unfortunate that we had the gap where no updates were tested and were just released

Reply:
The softfailure should be very specific to the circumstances, which in this case is the kernel version. This is the signature for the softfailure and must be specific, not generic.